docs/integrations/angular.rst

```diff
@@ -0,0 +1,48 @@
+Backbone
+========
+
+Installation
+------------
+
+Start by adding the ``raven.js`` script tag to your page. It should go **after** 
```

Comment:
But it only patches when calling `install()`, so if you want to do that, then you'd need to load and install sooner.